e2e/basics.spec.js

```diff
@@ -74,51 +74,63 @@ describe('Basic React Native and Interaction Support', () => {
   describe(':ios: Bridge API', () => {
     it('should call first track', async () => {
       await rnTestUtil.assertIosPixel(
-        { a: '2084764307', t: 'pressInTestEvent1' },
+        {
+          a: '2084764307',
+          t: 'pressInTestEvent1',
+          sprops: ['path', 'Basics', 'screen_name', 'Basics'],
```

Comment:
Do these get explicitly sorted by key at some point when it's converted from a map to an array?  I'm concerned these could get flickery down the line...

Reply:
Yea, I think so (either when it's converted to pixel format, or within the test utils).  I wouldn't be too concerned, since this will change once the iOS message format changes happen.